client/beefy/Cargo.toml

```
@@ -19,7 +19,7 @@ log = "0.4"
 parking_lot = "0.12.1"
 thiserror = "1.0"
 wasm-timer = "0.2.5"
-beefy-primitives = { version = "4.0.0-dev", path = "../../primitives/beefy" }
+beefy-primitives = { version = "4.0.0-dev", path = "../../primitives/beefy", package = "sp-beefy" }
```

nit:

since we're already touching tomls and also have companions for both Polkadot and Cumulus in this PR, why not go full rename from the get-go?

```suggestion
sp-beefy = { version = "4.0.0-dev", path = "../../primitives/beefy" }
```

---

That would incur touching the `.rs` files as well, right? I followed @athei 's suggestion from https://forum.parity.io/t/new-names-suggestions-for-squatted-crates/1513/2 of only changing the `Cargo.toml`s.

I went with this approach because it minimizes the amount of changes I have to do in this PR and the companions. However, I'm open to whatever approach is the most agreed upon. Changing the `.rs` files wouldn't be difficult, it'd just be more work. Also it might require more reviews from other teams.

Do you strongly prefer the full rename or is it acceptable to leave it like this?

---

Unless there's some hurry with this PR, I would prefer the full rename, otherwise you can kick the can down the road and someone will do it later; not blocking the PR on this :)

---

I think we *are* a bit in a hurry with it; since we want to release the crates ASAP.